### DISCUSSION TITLE
[Question] Combining Flet and Django - looking for example

### DISCUSSION THREAD

**Discussion by vindevoy** [flet-dev/flet#3300] [Q&A]
### Question

Hi,

A couple of weeks ago, as far as I remember, I saw a post by @ndonkoHenri , I think, who posted two links.  One contained an example of how to combine Flet and Django.  The other one was something with Django-Ninja.

The base of the code was that you start a standard Flet application.  Early in the code, the link was made with Django, using something like the Django settings that were imported and that made the links.  Then, you could simply query using Django's object.all().

I really would like to retrieve that example

Note: I'm not interested in libraries as django-flet. 

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Sorry, but that wasn't from me. 😅
No idea who posted that.

**Reply by sidviny** [reply]
But you also saw it ?  I was not dreaming ?  ;)

**Comment by baseplate-admin** [comment]
Probably this https://github.com/Marysia-Software-Limited/flet-django

**Reply by baseplate-admin** [reply]
If you dont want to use flet-django, you can copy thier Approach. Look into how they render flet and you can implement that in you application

**Reply by baseplate-admin** [reply]
Basically you want to return` ft.View`

**Reply by sidviny** [reply]
That is exactly what I don't want.

I don't want my entry point to be Django.  Here, you just program in Django again, just your interface becomes Flet instead of HTML.  It may look decent, but most likely for each page it will do a full reload, which is not needed/wanted when you have a top menu, a sidebar, ...

The example I saw, started from a Flet application, and then through importing someting of the Django settings classes, was able to talk query the database using the default Model.objects.all() method, for example.

**Reply by baseplate-admin** [reply]
Maybe you want to have every route point to the same view and then do the routing from flet?

Look into using `create-react-app` with `django`

**Reply by sidviny** [reply]
https://stackoverflow.com/questions/63726203/is-it-possible-to-use-fastapi-with-django

In this post, some code that comes close to what I'm looking for is shown.  Plus, django-ninja was used somewhere but that I'm still searching for.

**Reply by sidviny** [reply]
vindevoy and sidviny is just one person: me   ;)

vindevoy is my personal account, sidviny is my work account.
